frontend/src/store/slices/UserSlice.ts

```diff
@@ -0,0 +1,62 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import type { RootState } from "store/store";
+
+export interface User {
```

Comment:
WTF with this type.

username have any and erorr have undefined, when it's initialize several lines under.